src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java

```diff
@@ -845,46 +844,7 @@ private void maybeExecuteBashScripts()
                         logger.info("executing " + scriptPath +
                         " with arguments '" + absDirPath + "'");
 
-                        Process p = new ProcessBuilder(scriptPath.toString(),
-                            absDirPath.toString()).start();
-
-                        StatsDClient dClient
-                            = JigasiBundleActivator.getDataDogClient();
-                        if (dClient != null)
-                        {
-                            int returnValue;
-
-                            try
-                            {
-                                returnValue = p.waitFor();
-                            }
-                            catch (InterruptedException e)
-                            {
-                                logger.error("", e);
-                                returnValue = -1;
-                            }
-
-                            if (returnValue == 0)
-                            {
-                                dClient.increment(DD_ASPECT_SUCCESS);
-                                if (logger.isDebugEnabled())
-                                {
-                                    logger.debug("thrown stat: " +
-                                        DD_ASPECT_SUCCESS
-                                    );
-                                }
-                            }
-                            else
-                            {
-                                dClient.increment(DD_ASPECT_FAIL);
```

Comment:
This looks like a useful metric to have?

Reply:
Also note the behavior changes, we no longer call `p.waitFor()`

Reply:
This is something that is not used a lot, especially in our deployments. 
And the waitFor is just for the process finishing to get the output.